lighthouse-core/test/results/sample_v2.json

```diff
@@ -555,6 +555,26 @@
         ]
       }
     },
+    "maskable-icon": {
+      "id": "maskable-icon",
+      "title": "Manifest doesn't have a maskable icon",
+      "description": "A maskable icon ensures that the icon fills the entire shape when installing the app on a device. [Learn more](https://web.dev/maskable-icon/).",
+      "score": 0,
+      "scoreDisplayMode": "binary",
+      "explanation": "Failures: No manifest was fetched.",
```

Comment:
looks like this used to a multicheck audit so you'll probably need to `yarn update:sample-json` again :)